gtkm/static/script.js

```diff
@@ -0,0 +1,47 @@
+let githubUserData = {
+  githubName: "Name",
+  githubSurname: "Surname",
+  githubUsername: "namesurname",
+};
+
+const getUserData = function (userName, dataObject) {
+    fetch(`/github/stats/general_user`)
+      .then((response) => {
+        if (!response.ok){
+          throw new Error('HTTP response NOT OK');
+        }
+        return response.json();
+      })
+      .then((data) => {
+        if (!data){
+          throw new Error('Data is undefined');
+        }
+        else{
+          dataObject.githubName = data.name;
+          dataObject.githubSurname = data.surname;
+          dataObject.githubUsername = data.user_name;
+
+          let nameLabelList = document.querySelectorAll(".profile-name");
+          let usernameLabelList = document.querySelectorAll(".profile-username");
+
+          for (const element of nameLabelList){
+              element.textContent = dataObject.githubName + " " + dataObject.githubSurname;
+          }
+
+          for (const element of usernameLabelList){
+              element.textContent = dataObject.githubUsername;
+          }
+        } 
+      })
+      .catch(error => {
+        console.error('There has been a problem with fetch operation:', error);
+      });
+  };
+  
+const user = `kamilwil`;

```

Comment:
Czemu tu jest zahardcode'owana nazwa

Reply:
Kurczę, nie usunąłem jej po tym, jak zmienił się endpoint Jacka. Od razu wrzucam poprawkę.

Reply:
Oznaczam Cię jeszcze raz @pktiuk, bo wraz z tą poprawką dla porządku usunąłem też odpowiedni argument funkcji. U siebie sprawdziłem lokalnie, że działanie się nie zmieniło, ale ponieważ brak mi doświadczenia w "frontendowaniu", wolę nie podejmować pochopnych decyzji.

Reply:
Trochę dziwi mnie to twoje sprawdzanie xD

Powinieneś wiedzieć co robi twój kod bez sprawdzania (odpalania) ;)